src/malli/clj_kondo.cljc

```diff
@@ -103,7 +103,7 @@
 (defmethod accept :tuple [_ _ _ _] :seqable)
 (defmethod accept :multi [_ _ _ _] :any) ;;??
 (defmethod accept :re [_ _ _ _] :string)
-(defmethod accept :fn [_ _ _ _] :fn)
+(defmethod accept :fn [_ _ _ _] :any)
```

Comment:
I'm not sure if this is the only change that should be made, or if this breaks anything else? It seems relatively restricted to just the related issue.

Reply:
good catch. Function could return anything, not a function.